packages/yew/src/html/component/lifecycle.rs

```diff
@@ -205,8 +205,9 @@ where
         };
 
         if self.context.props != props {
+            let old_props = self.context.props.clone();
             self.context.props = props;
```

Comment:
Unnecessary clone. You can do something like `let old_props = std::mem::replace(&mut self.context.props, props);` to avoid it.

Reply:
Wow nice, thanks!

Done in ea192700

Reply:
(Fixed in 4b637217)